### PR TITLE
New subscription links

### PR DESCRIPTION
**Pull request by seriouspodcaster**
- Stitcher subscription link added
- Google Play subscription link added
- Formatting changed while showing subscription links
- Duration below media player removed
- Margin added to the media player

image

image

### CODE REVIEW

File: includes/class-ssp-frontend.php [hunk 1]
```diff
@@ -284,7 +284,7 @@ public function episode_meta( $episode_id = 0, $context = 'content' ) {
 
 			// Show audio player if requested
 			if( $show_player ) {
-				$meta .= '<div class="podcast_player">' . $this->media_player( $file, $episode_id ) . '</div>';
+				$meta .= '<div class="podcast_player" style="margin: 15px 0;">' . $this->media_player( $file, $episode_id ) . '</div>';
```
Comment (jonathanbossenger) [inline includes/class-ssp-frontend.php]: I'm not the biggest fan of inline styling, but I understand why this has been placed here. Would it be possible to add something like a 'player.css' file to be enquired on the front end of the site, for all custom CSS to be enqueued there. This way if any future changes need to be made we only need search from them in one place.

File: includes/class-ssp-frontend.php [hunk 2]
```diff
@@ -411,11 +415,27 @@ public function episode_meta_details ( $episode_id = 0, $context = 'content' ) {
 			}
 		}
 
+		$meta_display .= "<p>".__( 'Podcast:', 'seriously-simple-podcasting' )." ".$podcast_display."</p>";
+
 		$itunes_url = get_option( 'ss_podcasting_itunes_url', '' );
 		if ( ! empty( $itunes_url ) ) {
-			$meta_display .= $meta_sep . '<a href="' . esc_url( $itunes_url ) . '" title="' . __( 'Leave a review', 'seriously-simple-podcasting' ) . '" class="podcast-meta-itunes">' . __( 'Leave a review', 'seriously-simple-podcasting' ) . '</a>';
+			$subscribe_display .= '<a href="' . esc_url( $itunes_url ) . '" target="_BLANK" title="' . apply_filters( 'ssp_subscribe_link_name_itunes', __( 'iTunes', 'seriously-simple-podcasting' ) ) . '" class="podcast-meta-itunes">' . apply_filters( 'ssp_subscribe_link_name_itunes', __( 'iTunes', 'seriously-simple-podcasting' ) ) . '</a>';
```
Comment (jonathanbossenger) [inline includes/class-ssp-frontend.php]: Please keep all instances of _blank as lowercase, purely to keep uniformity in the code.

File: includes/class-ssp-frontend.php [hunk 3]
```diff
@@ -362,34 +362,38 @@ public function episode_meta_details ( $episode_id = 0, $context = 'content' ) {
...
+					$podcast_display .= '<a href="' . esc_url( $data ) . '" title="' . get_the_title() . ' " class="podcast-meta-download" download>' . __( 'Download file' , 'seriously-simple-podcasting' ) . '</a>';
 				break;
 
 				case 'new_window':
 					$play_link = add_query_arg( 'ref', 'new_window', $link );
-					$meta_display .= '<a href="' . esc_url( $play_link ) . '" target="_blank" title="' . get_the_title() . ' " class="podcast-meta-new-window">' . __( 'Play in new window' , 'seriously-simple-podcasting' ) . '</a>';
+					$podcast_display .= '<a href="' . esc_url( $play_link ) . '" target="_blank" title="' . get_the_title() . ' " class="podcast-meta-new-window">' . __( 'Play in new window' , 'seriously-simple-podcasting' ) . '</a>';
 				break;
 
 				case 'duration':
```
Comment (jonathanbossenger) [inline includes/class-ssp-frontend.php]: I'm not sure I understand the reason this has been left in?

File: includes/class-ssp-frontend.php (hunk 1 above)
Comment (seriouspodcaster) [reply includes/class-ssp-frontend.php]: Not a problem. I've created a player.css file to hold this instead :)

File: includes/class-ssp-frontend.php (hunk 3 above)
Comment (seriouspodcaster) [reply includes/class-ssp-frontend.php]: Should I rather remove the New Window link? The variable was changed though to separate the new subscribe links from the other player actions though.

File: includes/class-ssp-frontend.php (hunk 2 above)
Comment (seriouspodcaster) [reply includes/class-ssp-frontend.php]: Sure thing. Changed it to lowercase 👌

File: includes/class-ssp-frontend.php (hunk 3 above)
Comment (jonathanbossenger) [reply includes/class-ssp-frontend.php]: As discussed, please add the @todo comment, thanks

**Pull request merged**